PubSub/tests/Unit/SubscriptionTest.php

```diff
@@ -122,6 +122,10 @@ public function testDurations()
                 'messageRetentionDuration' => '1.1s',
                 'expirationPolicy' => [
                     'ttl' => '2.1s'
+                ],
```

Comment:
Can we add a test for empty retry policy as well as nil retry policy?

Reply:
I cover that in the system test. Do we need it for unit test as well?

Reply:
I see. SGTM.